Source/diablo.cpp

```diff
@@ -1032,199 +1018,200 @@ void __fastcall ReleaseKey(int vkey)
-							NetSendCmdString(1 << myplr, tempstr);
-							return;
-						case VK_F4:
-							PrintDebugQuest();
-							return;
+	else if(vkey == VK_F2) {
+	}
 #endif
-						case VK_F5:
-							v2 = 0;
-							goto LABEL_48;
-						case VK_F6:
-							v2 = 1;
-							goto LABEL_48;
-						case VK_F7:
-							v2 = 2;
-							goto LABEL_48;
-						case VK_F8:
-							v2 = 3;
-						LABEL_48:
-							if (spselflag)
-								SetSpeedSpell(v2);
-							else
-								ToggleSpell(v2);
-							return;
-						case VK_F9:
-							v3 = 0;
-						LABEL_59:
-							diablo_hotkey_msg(v3);
-							return;
-						case VK_F10:
-							v3 = 1;
-							goto LABEL_59;
-						case VK_F11:
-							v3 = 2;
-							goto LABEL_59;
-						case VK_F12:
-							v3 = 3;
-							goto LABEL_59;
-						case VK_UP:
-							if (stextflag) {
-								STextUp();
-							} else if (questlog) {
-								QuestlogUp();
-							} else if (helpflag) {
-								HelpScrollUp();
-							} else if (automapflag) {
-								AutomapUp();
-							}
-							return;
-						case VK_DOWN:
-							if (stextflag) {
-								STextDown();
-							} else if (questlog) {
-								QuestlogDown();
-							} else if (helpflag) {
-								HelpScrollDown();
-							} else if (automapflag) {
-								AutomapDown();
-							}
-							return;
-						case VK_PRIOR:
-							if (stextflag)
-								STextPrior();
-							return;
-						case VK_NEXT:
-							if (stextflag)
-								STextNext();
-							return;
-						case VK_LEFT:
-							if (automapflag && !talkflag)
-								AutomapLeft();
-							return;
-						case VK_RIGHT:
-							if (automapflag && !talkflag)
-								AutomapRight();
-							return;
-						case VK_TAB:
-							DoAutoMap();
-							return;
-						case VK_SPACE:
-							if (!chrflag) {
-								if (!invflag) {
-								LABEL_106:
-									helpflag = 0;
-									invflag = 0;
-									chrflag = 0;
-									sbookflag = 0;
-									spselflag = 0;
-									if (qtextflag && leveltype == DTYPE_TOWN) {
-										qtextflag = FALSE;
-										sfx_stop();
-									}
-									questlog = 0;
-									automapflag = 0;
-									msgdelay = 0;
-									gamemenu_off();
-									goto LABEL_110;
-								}
-								v4 = MouseX;
-								if (MouseX >= 480 || MouseY >= 352) {
-								LABEL_101:
-									if (!invflag && chrflag && v4 > 160 && MouseY < 352)
-										SetCursorPos(v4 - 160, MouseY);
-									goto LABEL_106;
-								}
-								SetCursorPos(MouseX + 160, MouseY);
-							}
-							v4 = MouseX;
-							goto LABEL_101;
-						}
-					}
-				}
-			}
+#ifdef _DEBUG
+	else if(vkey == VK_F3) {
+		if(pcursitem != -1) {
+			sprintf(
+				tempstr,
+				"IDX = %i  :  Seed = %i  :  CF = %i",
+				item[pcursitem].IDidx,
+				item[pcursitem]._iSeed,
+				item[pcursitem]._iCreateInfo);
+			NetSendCmdString(1 << myplr, tempstr);
 		}
+		sprintf(tempstr, "Numitems : %i", numitems);
+		NetSendCmdString(1 << myplr, tempstr);
+	}
+#endif
+#ifdef _DEBUG
+	else if(vkey == VK_F4) {

```

Comment:
I may be missing something, but cannot these `#ifdef _DEBUG` blocks be merged into one?

Reply:
Sometimes the Galaxy works in mysterious ways ^^